public/safe.webmanifest

```diff
@@ -15,5 +15,7 @@
     }
   ],
   "start_url": "/",
+  "theme_color": "#0FDA6D",
+  "background_color": "#121312",
```

Comment:
Will that work for both light and dark modes?

Reply:
Afaik this is still an experimental spec with not dark mode support. I added them because Lighthouse complained but looking more into it, we already have `<meta name="theme-color" ...>` for light and dark mode. I will revert this change.